Backend/.env.example

```diff
@@ -0,0 +1,7 @@
+HOST=http://localhost
+PORT=8080
+
+MONGO_URL=mongodb://mongoCont:27017/jasmine
+
+SERVER_HOST=10.8.100.245
```

Comment:
Why specifically this IP address?

Reply:
i added this for development purpose and this is Gen9 Server IP

Reply:
Ok, I see. Since this is intended to be an example, it is Ok to keep it as it is.